7_react/react1/week2/todo_list_app/src/App.jsx

```diff
@@ -0,0 +1,67 @@
+import { useEffect, useState } from "react";
+import reactLogo from "./assets/react.svg";
+import viteLogo from "/vite.svg";
+import "./App.css";
+import TodoList from "./components/todoList";
+import Form from "./components/form";
+import Header from "./components/header";
+
+function App() {
+  const [input, setInput] = useState("");
+  const [todos, setTodos] = useState([]);
+  //const [count, setCount] = useState(0);
+
+  const Timer = () => {
+    const [seconds, setSeconds] = useState(0);
+
+    useEffect(() => {
+      const interval = setInterval(() => {
+        setSeconds((seconds) => seconds + 1);
+      }, 1000);
+
+      return () => clearInterval(interval);
+    }, []);
+
+    return (
+      <div>
+        <p>You have used {seconds} seconds on this website</p>
+      </div>
+    );
+  };
```

Comment:
You extracted the timer into a separate component via a function. Great idea! 🧠 Since this might be a component that other pages would use too, it would also make sense to extract this code into another component file, like `Timer.jsx`. 😉